ml-worker.js

```
@@ -90,12 +90,18 @@ self.onmessage = async (e) => {
+
 // ---- Initialization ----
 async function initialize(msg) {
+  // FIX 1: Block any ortUrl from msg — ORT must be loaded locally only
+  if (msg.ortUrl) _bannedImport(msg.ortUrl);
+
   try {
-    // Import ONNX Runtime
+    // Import ONNX Runtime from vendored local copy only
     if (!ort) {
-      importScripts(msg.ortUrl || 'https://cdn.jsdelivr.net/npm/onnxruntime-web@1.17.0/dist/ort.min.js');
+      importScripts('/lib/ort.min.js'); // FIX 1: local vendored copy, never CDN
```

ESLint

[error] 'importScripts' is not defined.

(no-undef)